graph/api/api.go

```diff
@@ -0,0 +1,79 @@
+package api
```

Comment:
Personally, I would have used the "business" folder for this code.
No action really needed, it is just my preference.